### ISSUE TITLE
Failing IPv4 membership check

### ISSUE THREAD

**Issue by oguzhanunlu** [seancfoley/IPAddress#45]
Hi,

I'm trying to integrate IPAddress to my project but some unit tests fail unexpectedly. These unit tests are checking membership of an IP address in a CIDR. Let me share one;

```
IPAddressString cidr = new IPAddressString("217.152.89.223/27");
IPAddressString ip = new IPAddressString("217.152.89.198");
assertThat(cidr.contains(ip)).isTrue();
```

I expect this test to be true for any IP within [`217.152.89.192` - `217.152.89.223`] inclusive range and one of them is used here.

I use `com.github.seancfoley:ipaddress:5.3.1`.

Appreciate any help on resolving this. Thanks in advance.

**Comment by seancfoley** [comment]
Hello,

This is a duplicate of issue #26, issue #30, and issue #39.

This is explained in [this wiki entry](https://github.com/seancfoley/IPAddress/wiki/Code-Examples-1:-Creating,-Parsing,-Checking-Host-Address-Subnet,-Extracting-Address-Subnet-Info#parse-prefixed-ip-address-string-as-subnet-or-ip-address) and this [entry](https://github.com/seancfoley/IPAddress/wiki/Code-Examples-2:-Subnetting-and-Other-Subnet-Operations#check-if-subnet-contains-address-or-subnet) - see enclosingBlockContains() in particular.

In summary, 
217.152.89.223/27 is the same as 217.152.89.223.
So 217.152.89.223 contains 217.152.89.198 is false.

What you want is 217.152.89.192/27, which is the block of addresses from 217.152.89.192/27 to 217.152.89.223/27.
217.152.89.192/27 contains 217.152.89.198 is true.

**Comment by oguzhanunlu** [comment]
Thank you very much @seancfoley !
